src/components/Error/Error.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import { useState } from 'react';
 
-interface ErrorProps {}
+interface ErrorProps {
+  errorCode?: string;
+}
 
-export const Error: React.FC<ErrorProps> = ({}) => {
-  return <></>;
+export const Error: React.FC<ErrorProps> = ({
+  errorCode
+}) => {
+  const [ serverError ] = useState<ErrorProps["errorCode"]>(errorCode);
+
+  return (
+    <div>
+    {errorCode === 'Sorry, no markets found for that zip code. Please try again!' && <h2>Sorry, no markets found for that zip code. Please try again!</h2>}
+    { errorCode !== 'Sorry, no markets found for that zip code. Please try again!' && serverError === "500" && <h2>"Our Servers are down, please try again."</h2>}
```

Comment:
if serverError is assigned the value of errorCode (props) couldn't we just check if errorCode === 500 and remove line 11 (the hook establishing serverError)?

Reply:
Agreed, I'm not really sure what that's doing. I am also going to refactor utils so that the error code on 15 can be "fake404" and return the same h2 string